pkg/auth/model/model.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+option go_package = "github.com/treeverse/lakefs/auth/model";
+
+import "google/protobuf/timestamp.proto";
+
+package io.treeverse.lakefs.auth.model;
+
+message UserData {
+    int64 ID = 1;
```

Comment:
Why do we need ID in the KV world?
I think all queries are DisplayName/Username based, no?

Reply:
we need it for the conversion between the proto and the original, we will probably not use the ID while in proto model

Reply:
So I suggest we should remove it now? I don't see a reason for maintaining it when there are 0 paths using that field in the KV world and the DB world doesn't use this type.

Reply:
Actually, we decided something else in the [discovery](https://github.com/treeverse/lakeFS/pull/3405/files), so nm